web/src/engine/websites/Qiman5.ts

```diff
@@ -0,0 +1,39 @@
+import { Tags } from '../Tags';
+import icon from './Qiman5.webp';
+import { Chapter, DecoratableMangaScraper, Page, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON } from './../platform/FetchProvider';
+
+type APIChapters = {
+    data: {
+        name: string,
+        link : string
+    }[]
+}
+
+@Common.MangaCSS(/^{origin}\/[^.]+\.html$/, 'div.info h1.name_mh')
+@Common.MangasMultiPageCSS('/category/tags/6/page/{page}', 'div.item p.title a', 1, 1, 0, Common.AnchorInfoExtractor(true))
```

Comment:
Use an extractor that provides only the number required to identify the manga.

Reply:
But then i would have to write a custom FetchManga as MangaCSS doesnt allow to choose our own manga ID.